pkg/model/config/config.go

```diff
@@ -147,34 +147,58 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 	return true
 }
 
-// GVK contains information about scaffolded resources
-type GVK struct {
+// ResourceData contains information about scaffolded resources
+type ResourceData struct {
 	Group   string `json:"group,omitempty"`
 	Version string `json:"version,omitempty"`
 	Kind    string `json:"kind,omitempty"`
 
-	// CRDVersion holds the CustomResourceDefinition API version used for the GVK.
+	// CRDVersion holds the CustomResourceDefinition API version used for the ResourceData.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the GVK.
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Resource.
 	WebhookVersion string `json:"webhookVersion,omitempty"`
+
+	// Webhooks holds the Webhooks data
+	Webhooks *Webhooks `json:"webhooks,omitempty"`
+}
+
+// Webhooks contains information about scaffolded webhooks
+type Webhooks struct {
+	Defaulting bool `json:"defaulting,omitempty"`
+	Validation bool `json:"validation,omitempty"`
+	Conversion bool `json:"conversion,omitempty"`
 }
 
-// isEqualTo compares it with another resource
-func (r GVK) isEqualTo(other GVK) bool {
+// isGVKEqualTo compares it with another resource
+func (r ResourceData) isGVKEqualTo(other ResourceData) bool {
 	return r.Group == other.Group &&
 		r.Version == other.Version &&
 		r.Kind == other.Kind
 }
 
 // merge combines fields of two GVKs that have matching group, version, and kind,
 // favoring the receiver's values.
-func (r *GVK) merge(other GVK) {
+func (r *ResourceData) merge(other ResourceData) {
 	if r.CRDVersion == "" && other.CRDVersion != "" {
 		r.CRDVersion = other.CRDVersion
 	}
 	if r.WebhookVersion == "" && other.WebhookVersion != "" {
 		r.WebhookVersion = other.WebhookVersion
 	}
+	if other.Webhooks != nil {
+		if r.Webhooks == nil {
+			r.Webhooks = other.Webhooks
+		}
+		r.Webhooks.merge(*other.Webhooks)
+	}
+}
+
+// merge compares it with another webhook by setting each webhook type individually so existing values are
+// not overwritten.
+func (w *Webhooks) merge(other Webhooks) {
```

Comment:
You have the pointer so its easier to change this signature to receive a pointer that resolving the pointer to pass it to this function and it will perform better.
```suggestion
func (w *Webhooks) merge(other *Webhooks) {
```